Week1/homework/style.css

```diff
@@ -1 +1,20 @@
-/* add your styling here */
\ No newline at end of file
+#root ul {
+  display:flex;
+  flex-flow: row wrap;
+  list-style-type: none;
+  padding-inline-start: 0px;
```

Comment:
The `padding-inline-start` is a new property and isn't supported in Edge and IE, just using `padding` would work better is all browsers.